Toolchain/BuildClang.sh

```diff
@@ -248,7 +248,9 @@ popd
 
 SRC_ROOT=$($REALPATH "$DIR"/..)
 FILES=$(find \
+    "$SRC_ROOT"/AK \
```

Comment:
Where are we leaking `AK` includes into the C++ standard library headers? That looks like a layering issue.

Reply:
I  haven't checked whether there are any leaks. I just added it anyways to equalize both lists, to prevent another "why does this work on GCC but not on Clang" moment in the future.